### PR TITLE
Add Moki - Aggregator on TON

### PR DESCRIPTION
**Pull request by Ishnoor04**
Information About Moki

Moki is a decentralized exchange (DEX) aggregator on TON. It addresses key challenges in the decentralized finance (DeFi) ecosystem by aggregating liquidity from multiple DEXs into a single platform. Moki's primary function is to find the most efficient trading routes for users, ensuring they get the best possible prices with minimal slippage.

Website: https://moki.ag/
Telegram Channel: https://t.me/moki_ag
Twitter: https://x.com/moki_ag

### CODE REVIEW

**Comment by llamatester** [comment]
The moki adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators moki
    
    🦙 Running MOKI adapter 🦙
    ---------------------------------------------------
    Start Date:	Sun, 10 Nov 2024 00:00:00 GMT
    End Date:	Mon, 11 Nov 2024 00:00:00 GMT
    ---------------------------------------------------
    
    [Error: Request failed with status code 404] {
      url: 'https://api.leapwallet.io/ton-sor/api/v1/analytics/volumes?timestamp=1731283199000',
      method: 'GET',
      axiosError: 'Not Found',
      chain: 'ton'
    }

File: aggregators/moki/index.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const endpoint = 'ton-sor/api/v1/analytics/volumes';
+const start = 1727711103;// 30.09.2024
+
+interface IAPIResponse {
+    dailyVolume: string;
+    totalVolume: string;
+}
+
+const fetch: Fetch = async (timestamp) => {
+    const {dailyVolume, totalVolume}: IAPIResponse = await fetchURL(`${URL}${endpoint}?timestamp=${timestamp * 1000}`);
```
Comment (g1nt0ki) [inline aggregators/moki/index.ts]: @Ishnoor04 thanks for the PR, but is this the start timestamp of the 24 range or end timestamp? also, the api is not working?

**Review by g1nt0ki — APPROVED** [review]
.

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

File: aggregators/moki/index.ts (hunk 1 above)
Comment (Ishnoor04) [reply aggregators/moki/index.ts]: @g1nt0ki this is the 24 hour range end timestamp. Api is working now.

**Comment by llamatester** [comment]
The moki adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts aggregators moki
    
    🦙 Running MOKI adapter 🦙
    ---------------------------------------------------
    Start Date:	Wed, 13 Nov 2024 08:03:39 GMT
    End Date:	Thu, 14 Nov 2024 08:03:39 GMT
    ---------------------------------------------------
    
    TON 👇
    Backfill start time: 30/9/2024
    NO METHODOLOGY SPECIFIED
    Daily volume: 17.75 k
    Total volume: 51.29 k
    End timestamp: 1731571418 (2024-11-14T08:03:38.000Z)

File: aggregators/moki/index.ts (hunk 1 above)
Comment (g1nt0ki) [reply aggregators/moki/index.ts]: thanks for the PR

**Pull request merged**